### PR TITLE
Use flatcc for serialization of IPC messages.

### PR DESCRIPTION
**Pull request by pcmoritz**

### CODE REVIEW

File: src/plasma/plasma_store.c [hunk 1]
```diff
@@ -333,20 +337,20 @@ void seal_object(client *client_context,
...
+    object.data_size = entry->info.data_size;
+    object.metadata_size = entry->info.metadata_size;
     HASH_DELETE(handle, plasma_state->objects_notify, notify_entry);
     /* Send notifications to the clients that were waiting for this object. */
     for (int i = 0; i < utarray_len(notify_entry->waiting_clients); ++i) {
       client **c = (client **) utarray_eltptr(notify_entry->waiting_clients, i);
-      CHECK(plasma_send_reply((*c)->sock, &reply) >= 0);
-      CHECK(send_fd((*c)->sock, reply.object.handle.store_fd) >= 0);
+      CHECK(plasma_send_GetReply((*c)->sock, plasma_state->builder, &object_id, &object, 1) >= 0);
+      // CHECK(plasma_send_reply((*c)->sock, &reply) >= 0);
```
Comment (robertnishihara) [inline src/plasma/plasma_store.c]: Remove this?

File: src/plasma/CMakeLists.txt [hunk 2]
```diff
@@ -61,7 +64,22 @@ endif(APPLE)
...
+
+# Compile flatbuffers
+
+set(PLASMA_FBS_SRC "${CMAKE_SOURCE_DIR}/format/plasma.fbs")
+set(OUTPUT_DIR ${CMAKE_SOURCE_DIR}/format/)
+
+add_custom_target(gen_plasma_fbs ALL)
+
+add_custom_command(
+ TARGET gen_plasma_fbs
```
Comment (robertnishihara) [inline src/plasma/CMakeLists.txt]: two spaces

**Comment by pcmoritz** [comment]
Yay!

**Comment by mikkelfj** [comment]
Good to see flatcc being used - note that you likely want the more compact libflatccrt, not libflatcc, unless you also want to compile schema via the library. No harm done though.

**Comment by robertnishihara** [comment]
Thanks for the tip!

**Pull request merged**